specs/Standards/MultiToken/README.md

```diff
@@ -0,0 +1,539 @@
+    ///
+    /// This method is safe to call. It MUST NOT remove data.
+    ///
+    /// `amount` is sent as a string representing an unsigned 128-bit integer. If
+    /// omitted, contract MUST refund full `available` balance. If `amount` exceeds
+    /// predecessor account's available balance, contract MUST panic.
+    ///
+    /// If predecessor account not registered, contract MUST panic.
+    ///
+    /// MUST require exactly 1 yoctoNEAR attached balance to prevent restricted
+    /// function-call access-key call (UX wallet security)
+    ///
+    /// Returns the StorageBalance structure showing updated balances.
+    fn storage_withdraw(&mut self, token_ids:Vec<TokenId>, amount: Option<U128>) -> StorageBalance;
+
+    /// Unregisters the predecessor account and returns the storage NEAR deposit back.
+    ///
+    /// If the predecessor account is not registered, the function MUST return `false` without panic.
+    ///
+    /// If `force=true` the function SHOULD ignore account balances (burn them) and close the account.
+    /// Otherwise, MUST panic if caller has a positive registered balance (eg token holdings) or
+    ///     the contract doesn't support force unregistration.
+    /// MUST require exactly 1 yoctoNEAR attached balance to prevent restricted function-call access-key call
+    /// (UX wallet security)
+    /// Returns `true` iff the account was unregistered.
+    /// Returns `false` iff account was not registered before.
+    fn storage_unregister(&mut self, token_ids:Vec<TokenId>, force: Option<bool>) -> Vec<bool>;
+
+    fn storage_balance_bounds(&self, token_id:Vec<TokenId>, account_id: Option<AccountId>) -> StorageBalanceBounds;
+    fn storage_balance_of(&self, token_ids:Vec<TokenId>, account_id: AccountId) -> Option<StorageBalance>;
+}
+```
+
+### Metadata Trait
+```
+pub struct MultiTokenMetadata {
+    pub spec: String,              // required, essentially a version like "mt-1.0.0"
+    pub name: String,              // required, ex. "Mosaics"
+    pub symbol: Option<String>,            // ex. "MOSIAC"
+    pub icon: Option<String>,      // Data URL
+    pub base_uri: Option<String>, // Centralized gateway known to have reliable access to decentralized storage assets referenced by `reference` or `media` URLs
+    // supports metadata_uri interface that interpolates {id} in the string
+    pub decimals: Option<u8>, // Option to specify precision if required 
+    pub title: Option<String>, // ex. "Arch Nemesis: Mail Carrier" or "Parcel #5055"
+    pub description: Option<String>, // free-form description
+    pub media: Option<String>, // URL to associated media, preferably to decentralized, content-addressed storage
+    pub media_hash: Option<Base64VecU8>, // Base64-encoded sha256 hash of content referenced by the `media` field. Required if `media` is included.
+    pub copies: Option<u64>, // number of copies of this set of metadata in existence when token was minted.
```

Comment:
We call this `total supply` elsewhere; we call it `copies` here? Feels like unnecessary extra cognitive load to have these be different

Reply:
Copies could be removed, it's just some metadata field, it's unclear how the user would like to use this, maybe it's fixed supply maybe it's not. In fact we had some discussion here about representing supply in metadata, but that is of course strange for ft, but in reality, it's really developers choice. I'm happy to boot copies if people feel good about that . Let's bridge in the next meeting.